tests/Resources/Page/ODH/JupyterHub/JupyterLabLauncher.robot

```diff
@@ -232,11 +232,11 @@ Delete Folder In User Notebook
 
       # Verify that the jupyter notebook pod is running
       ${notebook_pod_name} =   Get User Notebook Pod Name  ${username}
-      OpenShiftLibrary.Search Pods    ${notebook_pod_name}  namespace=rhods-notebooks
-      #OpenShiftLibrary.Search Pods    ${notebook_pod_name}  namespace=redhat-ods-applications
+     # OpenShiftLibrary.Search Pods    ${notebook_pod_name}  namespace=rhods-notebooks
+      OpenShiftLibrary.Search Pods    ${notebook_pod_name}  namespace=redhat-ods-applications
 
-      ${output} =  Run   oc exec ${notebook_pod_name} -n rhods-notebooks -- rm -fr /opt/app-root/src/${folder}
-      #${output} =  Run   oc exec ${notebook_pod_name} -n redhat-ods-applications -- rm -fr /opt/app-root/src/${folder}
+      #${output} =  Run   oc exec ${notebook_pod_name} -n rhods-notebooks -- rm -fr /opt/app-root/src/${folder}
+      ${output} =  Run   oc exec ${notebook_pod_name} -n redhat-ods-applications -- rm -fr /opt/app-root/src/${folder}
```

Comment:
Why did you update the namespace to redhat-ods-applications? notebooks are spawned in rhods-notebooks now

Reply:
https://opendatascience-jenkins-csb-rhods.apps.ocp-c1.prod.psi.redhat.com/job/rhods-sanity/690/TestReport/ i was reveing this log and found this  
![Screenshot from 2022-08-05 16-28-00](https://user-images.githubusercontent.com/93319437/183065030-16296b4c-6bb6-47ac-be57-23bacd2e7d89.png)